plugins/incremental-ingestion-backend/src/router/paths.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const PROVIDER_CLEANUP = '/providers/cleanup';
+export const PROVIDER_HEALTH = '/providers/health';
+export const PROVIDER_BASE_PATH = '/providers/:provider';
```

Comment:
Note that it's risky to add both fixed and variable things under a root. This means you could never ever have a provider called `health` for example. Maybe these could be just `/cleanup`, `/health`, `/providers/:provider`?

Reply:
You're correct, and this was actually a mistake on my part. I meant to put them all into a path that clearly marked these routes as for incremental entity providers. I've corrected the mistake. They are now:

* `/incremental/cleanup`
* `/incremental/health`
* `/incremental/providers/:provider`

This ensures they won't impact anything else (a root of `providers` is too generic and opens the door for collisions), while also solving the potential for problems with a provider named "health" or "cleanup."